crawler/page_fetcher.py

```diff
@@ -48,4 +48,5 @@ def run(self):
         """
             Executa coleta enquanto houver páginas a serem coletadas
         """
-        pass
+        while self.obj_scheduler.has_finished_crawl() == False:
```

Comment:
Bacana demais Vinícius! Uma sugestão, oque acha de deixar assim? Acho que fica mais limpo:
```suggestion
        while not self.obj_scheduler.has_finished_crawl():
```